lib/contentTypeParser.js

```diff
@@ -90,6 +97,15 @@ ContentTypeParser.prototype.getParser = function (contentType) {
     }
   }
 
+  /* eslint-disable no-var */
+  for (var j = 0; j < this.parserRegExpList.length; j++) {
```

Comment:
Is this in a hot path? If not, I don't think we need to be so pedantic about `let` vs `var`.

Reply:
So you're suggesting going with let?
The path is executed if there is a cache mismatch.

Reply:
> The path is executed if there is a cache mismatch.

That does not tell me when the lookup happens. Is this at server boot? Or is it every incoming HTTP request?

Reply:
Every incoming HTTP request